vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/config/WriteConcernParser.java

```diff
@@ -11,51 +12,60 @@
 class WriteConcernParser {
   private final WriteConcern writeConcern;
 
-  public WriteConcernParser(JsonObject config) {
-    // Allow convenient string value for writeConcern e.g. ACKNOWLEDGED, SAFE, MAJORITY, etc
-    WriteConcern wc;
-    String wcs = config.getString("writeConcern");
-    if (wcs != null) {
-      wc = WriteConcern.valueOf(wcs);
-      if (wc == null) throw new IllegalArgumentException("Invalid WriteConcern " + wcs);
+  WriteConcernParser(ConnectionString connectionString, JsonObject config) {
```

Comment:
I would suggest to pull down the code change and using **gitk** to view and `ignore space changes`, that will make it clearer.